lesson 1/README.md

```diff
@@ -0,0 +1,363 @@
+# Lesson 0 - Github and an Intro to Java
+
+##  Github
+
+## Java
+
+### Getting started with Android Studio
+
+We will be using Android Studio as our IDE, even though (for this class) we won't actually be creating an Android app.
+
+1. Create a new Android Studio Project
+2. Name it (something?)
+3. Select Phone and Tablet, API 15 (This doesn't really matter for now)
+4. Empty Activity
+5. Finish
+
+This will create a basic Android application skeleton for you. We won't be using the skeleton, so we'll create our own Module.
+
+1. Click File > New > Module.
+2. Select Java Library and click Next.
+3. For Java class name, use "HelloWorld". Click Finish.
+4. You should now see a Java module (in a folder with whatever name you gave it) inside your Android project.
+
+### Hello World
+
+Copy paste into `HelloWorld.java` and read over the following code:
+
+```java
+public class HelloWorld {
+
+    public static void main(String[] args) {
+        System.out.println("Hello World!");
+    }
+
+}
+```
+
+A window button should appear next to the main method header. Click it to run your program!
+
+```
+Hello World!
+```
+
+Don't worry too much yet about what all the words in `public static void main(String[] args)` mean. For now, just remember that this is the **main method**, and that every java program must have a main method defined. This main method is what the Java Virtual Machine runs initially, and all subsequent code eventually stems from the main method.
```

Comment:
I honestly don't even know what it is. I assume its command line arguments you give it? It's not really worth explaining.